packages/components/src/internal/components/search/utils.ts

```diff
@@ -119,3 +125,181 @@ export function getSampleFinderQueryConfigs(
     }
     return configs;
 }
+
+export const SAMPLE_SEARCH_FILTER_TYPES_TO_EXCLUDE = [
+    Filter.Types.CONTAINS.getURLSuffix(),
+    Filter.Types.DOES_NOT_CONTAIN.getURLSuffix(),
+    Filter.Types.DOES_NOT_START_WITH.getURLSuffix(),
+    Filter.Types.STARTS_WITH.getURLSuffix(),
+    Filter.Types.CONTAINS_ONE_OF.getURLSuffix(),
+    Filter.Types.CONTAINS_NONE_OF.getURLSuffix(),
+];
+
+export const NEGATE_FILTERS = [
+    Filter.Types.NEQ_OR_NULL.getURLSuffix(),
+    Filter.Types.DATE_NOT_EQUAL.getURLSuffix(),
+    Filter.Types.NOT_IN.getURLSuffix(),
+];
+
+export const SAMPLE_SEARCH_FILTER_TYPES_SKIP_TITLE = [
+    Filter.Types.EQUAL.getURLSuffix(),
+    Filter.Types.DATE_EQUAL.getURLSuffix(),
+    Filter.Types.IN.getURLSuffix(),
+    Filter.Types.BETWEEN.getURLSuffix(),
+    ...NEGATE_FILTERS,
+];
+
+export function isBetweenOperator(urlSuffix: string): boolean {
+    return ['between', 'notbetween'].indexOf(urlSuffix) > -1;
+}
+
+export const FILTER_URL_SUFFIX_ANY_ALT = 'any';
+
+export function getSampleFinderFilterTypesForType(jsonType: JsonType): any[] {
+    const filterList = Filter.getFilterTypesForType(jsonType).filter(function (result) {
+        return SAMPLE_SEARCH_FILTER_TYPES_TO_EXCLUDE.indexOf(result.getURLSuffix()) === -1;
+    });
+    if (jsonType === 'date') {
+        filterList.push(Filter.Types.BETWEEN);
+        filterList.push(Filter.Types.NOT_BETWEEN);
+    }
+
+    const filters = [];
+
+    filterList.forEach(filter => {
```

Comment:
use `map` instead of `forEach` so you can just set the `filters` const directly?

Reply:
updated